src/routes/profil/+page.server.ts

```diff
@@ -0,0 +1,75 @@
+/** @type {import('./$types').Actions} */
+
+import { fail, redirect, error } from '@sveltejs/kit';
+
+import type { PageServerLoad, Actions } from './$types';
+import { supabase } from '$lib/auth';
+
+export const load = (async ({ params }) => {
+	return {
+		params,
+	};
+}) satisfies PageServerLoad;
+
+export const actions = {
+	default: async ({ request, cookies }) => {
+		const session_cookie = cookies.get('supabase-auth-token');
+
+		if (!session_cookie) {
+			throw error(500, {
+				code: 500,
+				message: 'No session cookie found',
+			});
+		}
+
+		const session = await supabase.auth
```

Comment:
Could you confirm that not getting `session` from `parent()` here is because `parent` is not available in form actions ?

Reply:
Yes, I confirm.